Tests/IntegrationTestProject/ProjectUnderTest/ProjectUnderTest/Foo.cs

```diff
@@ -1,7 +1,7 @@
 ﻿//-----------------------------------------------------------------------
 // <copyright file="Foo.cs" company="SonarSource SA and Microsoft Corporation">
 //   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
-//   Licensed under the MIT License. See License.txt in the project root for license information.
+//   Licensed under the MIT License. See License in the project root for license information.
```

Comment:
This still refers to the MIT License. Is that correct?
Also, you've deleted the license file from the project root so the comment isn't correct any more.

Reply:
Crap... It shouldn't be MIT.

The root does contain LICENSE.